website/_docs/cmd.clock.markdown

```diff
@@ -8,8 +8,13 @@ permalink: docs/cmd/clock.html
 
 Returns the current clock value for a watched root.
 
```

Comment:
Insert:

```
*Since 3.8.*

*The [capability](/watchman/docs/capabilities.html) name associated with this
enhanced functionality is `clock-sync-timeout`.*
```